app/Http/Controllers/TodoController.php

```diff
@@ -14,9 +14,30 @@ public function index(): JsonResponse
         return response()->json(['todos' => $todos]);
     }
 
-    public function create()
+    public function store(Request $request): JsonResponse
     {
-        // Show the form to create a new Todo
+        $validatedData = $request->validate([
+            'title' => 'required|string',
+            'description' => 'required|string',
+            'priority' => 'required|string',
+            'is_checked' => 'boolean',
+            'date' => 'date|nullable',
+            'user_id' => 'required|string',
+        ]);
+
+        // Create a new Todo instance and assign attributes
+        $newTodo = new Todo;
```

Comment:
You can use the built-in method of the model classes (`create`):
<img width="671" alt="CleanShot 2023-09-14 at 09 29 52@2x" src="https://github.com/Fekete-Balazs710/ToDo_Laravel/assets/74707175/3914e35b-bc54-4cd2-81c1-da607dc8817e">